### PR TITLE
update the documentation for zh-cn

### PR DESCRIPTION
**Pull request by yorkie**
Just updates for Chinese Simplification Version.

### CODE REVIEW

**Review by LanFly — APPROVED** [review]

File: zh-cn/yodaos-source/00-source-tree.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+Lunch menu... pick a image style:
+
+1. debug(root filesystem bin is not striped)（目前受限于 k18 flash 的大小，debug 模式下也是不带符号表）
+2. release(root filesystem bin is striped)
+```
+
+./build.sh 添加 -d 选项：
+
+```shell
+$ ./build.sh -p leo_k18_universal_node -n openwrt-leo-k18-universal -f leo-k18-universal  -j32 -r -d
```
Comment (FeelyChau) [inline zh-cn/yodaos-source/00-source-tree.md:99]: 如果是 k18 的话, 需要增加一个参数 `-x k18`, 用以明确指定编译平台.

**Review by FeelyChau — CHANGES_REQUESTED** [review]

File: zh-cn/yodaos-source/00-source-tree.md (hunk 1 above)
Comment (yorkie) [reply zh-cn/yodaos-source/00-source-tree.md:99]: 这里暂时先不改动。

File: zh-cn/INTRO.md [hunk 2]
```diff
@@ -1,25 +1,41 @@
...
+- **应用框架（Application Framework）**承担了承上启下的作用，它负责定义 [YodaOS Application][] 以及管理其安装、执行和生命周期等过程，并提供接口让应用开发者使用系统框架所提供的能力。
 
-### License
+本书主要面向两类读者：
 
-Apache v2.0
+- 如果你想作为应用开发者，为 YodaOS 开发语音交互应用，那么可以从 [开发者指北][] 开始。
+- 如果你想深度定制，那么可以从 [YodaOS Source][] 开始。
 
+[Rokid]: https://www.rokid.com
```
Comment (legendecas) [inline zh-cn/INTRO.md]: No use cases?

File: zh-cn/yodaos-source/00-source-tree.md [hunk 3]
```diff
@@ -0,0 +1,168 @@
+## 下载代码
+
+YodaOS 的代码使用 GitHub 和 Gerrit 分别进行管理，开发者可以参考[官方下载教程](https://github.com/yodaos-project/yodaos#download-source)具体了解。
+
+## 目录介绍
+
+接下来先按照字母顺序，介绍下一级目录如下：
+
+| 子目录         | 是否目录 | 描述 |
```
Comment (legendecas) [inline zh-cn/yodaos-source/00-source-tree.md]:
```suggestion
| 子节点         | 是否目录 | 描述 |
```

File: zh-cn/yodaos-source/00-source-tree.md [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+* `Makefile` 在顶层目录执行make命令的入口文件。
+* `rules.mk` 定义了Makefile中使用的一些通用变量和函数。
+* `Config.in` 在 `include/toplevel.mk` 中我们可以看到，这是和 `make menuconfig` 相关联的文件。
+* `feeds.conf.default` 是下载第三方一些软件包时所使用的地址。
+   
+##### 生成目录
+
+* `./build_dir` 在前面的原始目录中，我们提到了 host 工具，toolchain 工具还有目标文件。OpenWrt 将在这个目录中展开各个软件包，进行编译，所以这个文件夹中包含3个子文件夹:
+  * `./host` 在该文件夹中编译主机使用的工具软件。
+  * `./toolchain-XXX` 在该文件夹中编译交叉工具链。
```
Comment (legendecas) [inline zh-cn/yodaos-source/00-source-tree.md]:
```suggestion
  * `./toolchain-<target-name>` 在该文件夹中编译交叉工具链。
```

File: zh-cn/yodaos-source/00-source-tree.md [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+  * `./target-XXX` 在此编译目标平台的目标文件，包括各个软件包和内核文件。
```
Comment (legendecas) [inline zh-cn/yodaos-source/00-source-tree.md]:
```suggestion
  * `./target-<target-name>` 在此编译目标平台的目标文件，包括各个软件包和内核文件。
```

File: zh-cn/yodaos-source/00-source-tree.md [hunk 6]
```diff
@@ -0,0 +1,168 @@
...
+  * `./bin` 保存编译完成后的二进制文件，包括完整的 bin 文件,所有的 ipk 文件。
+* `./staging_dir` 用于保存在 build_dir 目录中编译完成的软件，所以这里也和 build_dir 有同样的子目录结构。比如，在 target-XXX 文件夹中保存了目标平台编译好的头文件，库文件。在我们开发自己的 ipk 文件时，编译过程中，预处理头文件，链接动态库，静态库都是到这个子文件夹中。
```
Comment (legendecas) [inline zh-cn/yodaos-source/00-source-tree.md]: Ditto. Use an expressive name.

File: zh-cn/yodaos-source/00-source-tree.md [hunk 7]
```diff
@@ -0,0 +1,168 @@
...
+##### 编译基础
+
+- 首先通过 `cd openwrt` 进入到 openwrt 目录
+- 然后拷贝你所需的 defconfig 到 `.config` 覆盖后，执行 `make defconfig`
+- 最后通过 `make V=s` 编译即可
+
+##### 使用 menuconfig 生成配置
+
+- 运行 `make menuconfig` 完成配置修改
+- 运行 `./scripts/diffconfig.sh > xxxx_defconfig` 生成脚本
```
Comment (legendecas) [inline zh-cn/yodaos-source/00-source-tree.md]: Ditto. Use an expressive name.

File: zh-cn/yodaos-source/00-source-tree.md [hunk 8]
```diff
@@ -0,0 +1,168 @@
...
+### frameworks
+
+框架目录中的结构相对比较复杂，这里只做第一层目录的介绍，如需了解更多可以在代码中具体查看：
+
+| 子目录     | 描述 |
+|-----------|------|
+| aial      | 算法库、神经网络库、语音激活库、离线自然语言处理等 |
+| native    | 系统功能库与服务，包括多媒体、蓝牙、音量等 |
+| vm        | 应用虚拟机，包括 Node.js 和 ShadowNode |
+| jsruntime | 上层 JavaScript 运行时代码 |
```
Comment (legendecas) [inline zh-cn/yodaos-source/00-source-tree.md]:
```suggestion
| jsruntime | JavaScript 应用运行时代码 |
```

File: zh-cn/yodaos-source/00-source-tree.md [hunk 9]
```diff
@@ -0,0 +1,168 @@
...
+
+### yodart 介绍
+
+[yodaos-project/yodart][] 是 [YodaOS][] 的上层应用层框架，下面对其做简单介绍。
```
Comment (legendecas) [inline zh-cn/yodaos-source/00-source-tree.md]:
```suggestion
[yodaos-project/yodart][] 是 [YodaOS][] 的应用层框架，下面对其做简单介绍。
```

File: zh-cn/yodaos-source/00-source-tree.md [hunk 10]
```diff
@@ -0,0 +1,168 @@
...
+| apps                | 内置本地应用，包括蓝牙音乐、音量控制、配网等 |
+| apps/cloudappclient | 云端技能的本地客户端，处理云端下发的逻辑 |
+| include             | 构建时依赖的项目头文件 |
+| packages            | 通用模块接口，用于调用系统底层服务 |
+| res                 | 资源文件包括灯光和音效 |
+| runtime             | 核心代码与服务 |
+| test                | 单元测试 |
+| tools               | 调试工具 |
+
+由于 [yodaos-project/yodart][] 大部分是脚本语言，因此可以直接安装到设备端运行：
```
Comment (legendecas) [inline zh-cn/yodaos-source/00-source-tree.md]: 是否需要注明存在 native package?

File: zh-cn/yodaos-source/voice-interactive/voice-interface.md [hunk 11]
```diff
@@ -1,157 +0,0 @@
-# voice-interface flora消息定义
```
Comment (legendecas) [inline zh-cn/yodaos-source/voice-interactive/voice-interface.md:1]: Why had this file been removed?

File: zh-cn/yodaos-source/00-source-tree.md (hunk 10 above)
Comment (yorkie) [reply zh-cn/yodaos-source/00-source-tree.md]: What do you mean by the native package? The C/C++ application?

File: zh-cn/yodaos-source/00-source-tree.md (hunk 3 above)
Comment (yorkie) [reply zh-cn/yodaos-source/00-source-tree.md]: Is there any reason to change "目录" to "节点"?

File: zh-cn/yodaos-source/00-source-tree.md (hunk 8 above)
Comment (yorkie) [reply zh-cn/yodaos-source/00-source-tree.md]: "YodaOS 应用框架代码" would be better?

File: zh-cn/INTRO.md (hunk 2 above)
Comment (yorkie) [reply zh-cn/INTRO.md]: Removed :)

File: zh-cn/yodaos-source/voice-interactive/voice-interface.md (hunk 11 above)
Comment (yorkie) [reply zh-cn/yodaos-source/voice-interactive/voice-interface.md:1]: Weird, I should have removed this file on latest revision, could you check again?

File: zh-cn/yodaos-source/customization/03-light.md [hunk 12]
```diff
@@ -55,13 +56,13 @@
...
-### 用户灯效文件定义
+#### 用户灯效文件定义
 
-所有非系统灯效都被定义为用户灯效。也就是说，即使你的文件在系统目录下，但是不在`config.json`文件中声明，它还是用户灯效。
+所有非系统灯效都被定义为用户灯效。也就是说，即使你的文件在系统目录下，但是不在：`config.json` 文件中声明，它还是用户灯效。
 
 用户灯效无需定义，也没有预先声明的优先级，所以它的优先级是运行时，由调用方动态指定的。
 
-用户灯效同样也有一个layer层。只不过这个层是数字越小，优先级越低，默认是0。恢复的时候，从最大层开始查找，直到0层。最大层数由lightd服务里面配置。如果调用的时候超出了范围，则会自动限定在范围内。比如超出了最大范围，则默认为最大层数。
+用户灯效同样也有一个 Layer。只不过这个层是数字越小，优先级越低，默认是0。恢复的时候，从最大层开始查找，直到0层。最大层数由lightd服务里面配置。如果调用的时候超出了范围，则会自动限定在范围内。比如超出了最大范围，则默认为最大层数。
```
Comment (FeelyChau) [inline zh-cn/yodaos-source/customization/03-light.md]:
```suggestion
用户灯效同样也有一个 Layer。只不过这个层是数字越小，优先级越低，默认是0。恢复的时候，从最大层开始查找，直到0层。最大层数在 lightd 服务中配置。如果调用的时候超出了范围，则会自动限定在范围内。比如超出了最大范围，则默认为最大层数。
```

File: zh-cn/yodaos-source/customization/03-light.md [hunk 13]
```diff
@@ -23,10 +24,10 @@
...
 
-1. 系统灯效文件。
-2. 用户灯效文件。
+- 系统灯效文件。
+- 用户灯效文件。
 
-### 系统灯效文件定义
+#### 系统灯效文件定义
 
 系统灯效在 `${LIGHT_SOURCE}/config.json` 中配置。此文件的内容是json对象。key保存系统灯效的相对路径，以`config.json`文件为相对路径，值为该灯光的优先级。系统灯效的优先级数字越小，表示优先级越高，也就是0最高。
```
Comment (FeelyChau) [inline zh-cn/yodaos-source/customization/03-light.md]:
```suggestion
系统灯效在 `${LIGHT_SOURCE}/config.json` 中配置。此文件的内容是 json 对象。key 保存系统灯效的相对路径，以`config.json`文件为相对路径，值为该灯光的优先级。系统灯效的优先级数字越小，表示优先级越高，也就是0最高。
```

File: zh-cn/yodaos-source/customization/03-light.md [hunk 14]
```diff
@@ -70,68 +71,82 @@
...
 
-### lightd 架构
+#### 架构
 
-lightd 整体分为2部分：
+lightd 整体分为两部分：
 
- - 一是上层 JavaScript 应用框架，负责抽象硬件 api 和管理资源。
- - 二是底层硬件渲染库，负责将上层应用设置的灯效最终渲染到硬件上。
+- 一是上层 JavaScript 应用框架，负责抽象硬件 api 和管理资源。
```
Comment (FeelyChau) [inline zh-cn/yodaos-source/customization/03-light.md]:
```suggestion
- 上层 JavaScript 应用框架，负责抽象硬件 api 和管理资源。
```

File: zh-cn/yodaos-source/customization/03-light.md [hunk 15]
```diff
@@ -70,68 +71,82 @@
...
+- 二是底层硬件渲染库，负责将上层应用设置的灯效最终渲染到硬件上。
```
Comment (FeelyChau) [inline zh-cn/yodaos-source/customization/03-light.md]:
```suggestion
- 底层硬件渲染库，负责将上层应用设置的灯效最终渲染到硬件上。
```

**Review by FeelyChau — APPROVED** [review]

File: zh-cn/yodaos-source/00-source-tree.md (hunk 3 above)
Comment (legendecas) [reply zh-cn/yodaos-source/00-source-tree.md]: `build.sh` is not a directory.

File: zh-cn/yodaos-source/00-source-tree.md (hunk 10 above)
Comment (legendecas) [reply zh-cn/yodaos-source/00-source-tree.md]: Packages like `@yodaos/speech-synthesis`.

**Review by FeelyChau — APPROVED** [review]
LGTM.

File: zh-cn/yodaos-source/00-source-tree.md (hunk 10 above)
Comment (yorkie) [reply zh-cn/yodaos-source/00-source-tree.md]: Ok, tweaked to describe it's relationships from the source with target in latest revision :)

**Comment by yorkie** [comment]
@legendecas @FeelyChau Updated :)

File: zh-cn/yodaos-source/voice-interactive/voice-interface.md (hunk 11 above)
Comment (legendecas) [reply zh-cn/yodaos-source/voice-interactive/voice-interface.md:1]: Still marked as removed?

Comment (yorkie) [reply zh-cn/yodaos-source/voice-interactive/voice-interface.md:1]: Sorry, I'll revert this :p

**Review by legendecas — APPROVED** [review]

**Pull request merged**